app/tests/common_tests/test_link_visibility.py

```diff
@@ -11,20 +11,11 @@
 )
 def test_admins_see_links(view, client, two_challenge_sets):
     url = reverse(
-        "pages:home",
+        "pages:admin",
```

Comment:
Hmm, is it only to support this test? Then maybe what we need to do is re-write the tests to test each of the challenge admin views for permissions. I can show you some good examples that we have.